src/components/integrated/OakLessonTopNav/OakLessonTopNav.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { ReactNode } from "react";
+import styled from "styled-components";
+
+import {
+  OakBox,
+  OakFlex,
+  OakHeading,
+  OakIconProps,
+  OakSpan,
+} from "@/components/base";
+import { OakRoundIcon } from "@/components/ui";
+import { getBreakpoint } from "@/styles/utils/responsiveStyle";
+
+type LessonSectionName = "intro" | "starter-quiz" | "video" | "exit-quiz";
+
+const StyledMobileSummary = styled(OakSpan)`
+  @media (min-width: ${getBreakpoint("small")}px) {
+    display: none;
+  }
+`;
+
+export type OakLessonTopNavProps = {
+  lessonSectionName: LessonSectionName;
+  /**
+   * Slot to render `OakBackLink` or similar
+   */
+  backLinkSlot: ReactNode;
+  heading: ReactNode;
+  /**
+   * Displayed at the mobile breakpoint where the counter is not rendered.
+   * Provides alternative content for the counter/progress in the lesson.
+   */
+  mobileSummary: ReactNode;
```

Comment:
This leaves me feeling a bit icky but the design calls for the counter to be hidden on mobile and for some extra text to be displayed below the heading instead. 

<img width="854" alt="Screenshot 2024-01-16 at 15 52 47" src="https://github.com/oaknational/oak-components/assets/122096/8c431297-f8e7-46d5-b620-c3595cdeab0f">
<img width="347" alt="Screenshot 2024-01-16 at 15 52 43" src="https://github.com/oaknational/oak-components/assets/122096/cde39c45-3406-4c92-845f-b0236bd1ef2c">

I'm hiding the counter completely with `display: none` on mobile and hiding the `mobileSummary` with `display: none` when above mobile size. It doesn't seem to make sense to display both to AT at the same time so this seems like a sensible approach to me.

Might be a question for @rightsaidjames but can anyone see any pitfalls with this approach?

Reply:
This seems like the correct approach to me - AT should (and does!) respect `display: none`* and media queries, so giving them an equivalent experience rather than a dedicated one is clearly the right solution in this scenario.

*_and `block`, `inline`, `inline-block` etc._